src/components/Tooltip/Tooltip.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+import { textKilo } from '../../styles/style-helpers';
+
+const baseStyles = css`
+  label: tooltip;
+  position: relative;
+  display: inline-block;
+
+  &:hover {
+    > span {
+      visibility: visible;
+      opacity: 1;
+    }
+  }
+`;
+
+const arrowAligns = theme => ({
+  center: `left: 50%`,
+  right: `left: ${theme.spacings.mega}`,
+  left: `right: ${theme.spacings.mega}`
+});
```

Comment:
Although it's a neat solution, this won't work with our static style extraction.